nff/tests/test_ase.py

```diff
@@ -5,6 +5,8 @@
 import numpy as np
 from ase import Atoms
 
+import pytest
+
```

Comment:
I never looked into it too much, because it "just works" and pytest is the most commonly used testing system, but to my understanding the advantages are:
 
  1) Execution is quite simple, pytest follows an easy rule which functions in which files to execute and executes just these functions, so there is no extra code in the test files necessary
  2) Pytest integrates very well with easily available coverage reports
  3) In our case we have the CPU/GPU problem with most of the code base and pytest offers a not super easy, but relatively easy way to adjust variables in tests based on user settings when executing pytest. So in this PR I also add that you can do `pytest --device cpu` or `pytest --device cuda` to execute the tests either with CPU or GPU. CPU is the default, because it is easier on the CI. So far this is not documented, but I would focus on documentation / readme after linting the code base, but definitely worth putting on a list